.travis.yml

```diff
@@ -51,6 +51,12 @@ matrix:
       os: osx
       env: BOINC_TYPE=manager-osx
     - language: android
+      android:
+        components:
+            - build-tools-28.0.2
+            - android-26
```

Comment:
Hm, can you envision a way to extract those details from https://github.com/BOINC/boinc/blob/master/android/BOINC/app/build.gradle#L49 and use/integrate them here? Can you include other files (with variable definitions) in the travis config? The idea here again would be to avoid redundant config settings that have to be kept in sync manually otherwise - somethings that's asking for inconsistencies and thus problems.

Reply:
I'm afraid no. @ChristianBeer could know more about this

Reply:
I don't know a way to include other files into the travis config. I guess we have to put a comment in the gradle file to remind whoever changes it to also change the CI system dependency. Of course this point is mood if the gradle file is changed by a UI like Android Studio.